### ISSUE TITLE
Release 1.7.8.4

### ISSUE THREAD

**Issue by MatShir** [PrestaShop/PrestaShop#27219]
This issue tracks the status of the PrestaShop 1.7.8.4 release

### Information

Q                                    | A
-----------------------|---------------
**Type**          | Patch version
**Est. Build date**  |  Februray 23, 2022
**Milestone** | [1.7.8.4](https://github.com/PrestaShop/PrestaShop/milestone/80)
**Kanban** | [1.7.8.4](https://github.com/PrestaShop/PrestaShop/projects/23)
**Est. Delivery date**          |  March 9, 2022
**Release manager tech**        |  @NeOMakinG 
**Release manager product**     |  @fabienvallon 
### Current status

- [x] Development
- [x] Build
- [x] QA validation
- [x] Release
- [x] Available for upgrade
- [x] Docker image

**Comment by enginteknoloji** [comment]
1.7.8.3 kurulumundan sonra içine Farklı bir Tema Kurulumu Yapılıp Kurulan Tema sorunsuz Çalıştırdıktan Sonra..Kurulan Temadan sepete ürün Eklendiğinde. Ürün Eklemede Sepette Görünümünde Bir Sorun Yok Ancak Sepetteki Ürünü Kaldır Butonuna Bastıktan Sonra Sepette Ürün Duruyor..Sayfa refresh Yenilendikten Sonra Sepetteki ürün Siliniyor..Bütün Temalarda Büyle Bu Düzeltmeyide Yaparmısınız

**Comment by NeOMakinG** [comment]
Hello,

Please use the English language, I didn't understand anything.

Create a proper issue dedicated to your problem, so we are able to reproduce the bug.

Thanks in advance!

**Comment by enginteknoloji** [comment]
1.7.8.3 After the installation, a different Theme is Installed and the Installed Theme is run again.. When the product is added to the cart from the Installed Theme. There is no problem in adding the product in the basket. However, the product in the basket remains in the basket after clicking the Remove button. After the page is refreshed, the product in the basket is deleted.. Can you make this correction in all the themes?
testing....https://hepsioffis.com/women/2-9-brown-bear-printed-sweater.html#/1-boyut-s

**Comment by MatShir** [comment]
Hi @enginteknoloji,
I have tested on my side, the default theme seems to work accordingly. We can't reproduce the issue on our side. The bug may come from your theme. I suggest consulting with your module developer. Have a good day :)

**Comment by enginteknoloji** [comment]
Thnks

**Comment by NeOMakinG** [comment]
Build just sent to the QA

**Comment by Robin-Fischer-PS** [comment]
Hello there :)

**Final report of QA on the 1.7.8.4 build 1 :**

**No 1.7.8.4 regressions have been found.**

- An already found regression on 1.7.8.3 is a bit more complex : (see last comments on https://github.com/PrestaShop/PrestaShop/issues/27684#issuecomment-1054103933)
- A minor regression since 1.7.8.3 has been found : https://github.com/PrestaShop/PrestaShop/issues/27799
- A minor regression since 1.7.8.0 has been found : https://github.com/PrestaShop/PrestaShop/issues/27794
- A major bug (not regression, already there in 1.7.7.8) has been found : https://github.com/PrestaShop/PrestaShop/issues/27807
- An improvement is also suggested : https://github.com/PrestaShop/PrestaShop/issues/27823

Some others bugs (no 1.7.8 regressions) were found, the issues are yet to be created. 
Finally, we found some bugs linked to welcome module (no regression), but since the module will be archived, we will not create issues for these bugs.

In conclusion, it's a GO to continue the release ! QA :heavy_check_mark:

**Comment by NeOMakinG** [comment]
Waiting for the release note, and then we can close this issue!

**Comment by NeOMakinG** [comment]
1.7.8.4 is released!
